apps/voting/app/src/App.js

```diff
@@ -68,13 +83,29 @@ function App() {
           />
           {selectedVote ? (
             <VoteDetail
+              selectVote={selectVote}
               vote={selectedVote}
               onBack={handleBack}
               onVote={actions.vote}
               onExecute={actions.execute}
             />
           ) : (
-            <Votes votes={votes} selectVote={selectVote} />
+            <Votes
+              votes={votes}
+              selectVote={selectVote}
+              filteredVotes={filteredVotes}
+              voteStatusFilter={voteStatusFilter}
+              handleVoteStatusFilterChange={handleVoteStatusFilterChange}
+              voteOutcomeFilter={voteOutcomeFilter}
+              handleVoteOutcomeFilterChange={handleVoteOutcomeFilterChange}
+              voteTrendFilter={voteTrendFilter}
+              handleVoteTrendFilterChange={handleVoteTrendFilterChange}
+              voteAppFilter={voteAppFilter}
+              handleVoteAppFilterChange={handleVoteAppFilterChange}
+              voteDateRangeFilter={voteDateRangeFilter}
+              handleVoteDateRangeFilterChange={handleVoteDateRangeFilterChange}
+              handleClearFilters={handleClearFilters}
```

Comment:
Not a big fan of this; why not just use the hook directly in `Votes`?

Reply:
Because filter state gets lost when selecting a vote and coming back to the list, this way it persists and the user can come back to a filtered list of votes.